components/test/testPolyfills.ts

```diff
@@ -21,6 +21,9 @@ window.requestAnimationFrame = function (cb: FrameRequestCallback) {
 
 window.loadTimeData = {
   getString (key) {
+    if (key === 'braveWalletLedgerBridgeUrl') {
+      return 'chrome-untrusted://ledger-bridge'
+    }
```

Comment:
Needed to so that LEDGER_BRIDGE_URL is an actually URL for the tests, not "braveWalletLedgerBridgeUrl"